README.md

```diff
@@ -1,16 +1,62 @@
 # packer-images
 
-A repository containing a number of packer build configurations for AWS & Azure.
+A repository containing a number of packer build configurations to generate Jenkins agent templates for all the Jenkins controllers
+of the jenkins-infra project, such as ci.jenkins.io, infra.ci.jenkins.io, etc.
 
-## Requirements
+## Contributing
```

Comment:
better to make a separate CONTRIBUTING.md file that will be shown automatically by github on PRs

Reply:
Do you mind if I do this as a subsequent PR? The 40 min wait time for a given PR is a bit much 😅

Reply:
belongs in a separate PR imo

Reply:
https://github.com/jenkins-infra/packer-images/pull/88

Reply:
Fair, I've added the change here as build was failing.

Reply:
i meant that these should be separate PRs as CONTRIBUTING should be discussed and agreed whilst fixing the build and getting the Azure agents working is separate.

i.e. merge your fixes here, but keep the ways of working stuff separate